.circleci/config.yml

```
@@ -160,7 +160,8 @@ jobs:
           name: build the docs
           command: |
             . env/bin/activate
-            make -C docs/ html
+            sudo apt update && sudo apt install doxygen
```

To fix `dot` errors:
```suggestion
            sudo apt update && sudo apt install doxygen graphviz
```